ci/variables/env

```diff
@@ -0,0 +1,2 @@
+DEFAULT_TERRAFORM_VERSION=1.4.4
+DEFAULT_KITCHEN_TERRAFORM_TAG=2.0.1
```

Comment:
`DEFAULT_TERRAFORM_VERSION` is not used anywhere. Do we need it?

I think for one variable we should probably keep it inline in the file and remove this file altogether

Reply:
DEFAULT_TERRAFORM_VERSION is removed

Reply:
Should the entire file be removed instead? I don't see it storing any more variables in the near future, but happy to hear your thoughts.

Reply:
ci/variables/env is removed and moved the variable inside the script.